src/Model/TransformerInterface.php

```diff
@@ -22,7 +22,7 @@
  *   id: int|string|null,
  *   name?: string|null,
  *   enabled: boolean,
- *   position: int|null,
+ *   position: int|string|null,
```

Comment:
It didn't followed the whole but since you're doing a is numeric check, should it be `numeric-string` here ?

Reply:
numeric-string is more restrictive than string. IMO we should try to see if the string is numeric, so we cast it, or is not numeric, so we not setPosition.